tabular/src/autogluon/tabular/models/fasttext/fasttext_model.py

```diff
@@ -0,0 +1,158 @@
+__all__ = ["FastTextModel"]
+
+
+import logging
+import tempfile
+from collections import defaultdict
+
+import numpy as np
+import pandas as pd
+from pandas.api.types import is_numeric_dtype
+
+from autogluon.core.utils.loaders import load_pkl
+from autogluon.core.utils.savers import save_pkl
+
+from ...features.feature_metadata import FeatureMetadata
+from ...features.feature_metadata import R_CATEGORY, R_OBJECT,  S_TEXT
+
+from ...constants import BINARY, MULTICLASS
+from ..abstract.abstract_model import AbstractModel
+from .hyperparameters.parameters import get_param_baseline
+
+logging.basicConfig(
+    format="%(asctime)s: [%(funcName)s] %(message)s", level=logging.INFO
+)
+logger = logging.getLogger(__name__)
+
+
+def try_import_fasttext():
+    try:
+        import fasttext
+
+        _ = fasttext.__file__
+    except Exception:
+        raise ImportError('Import fasttext failed. Please run "pip install fasttext"')
+
+
+class FastTextModel(AbstractModel):
+    model_bin_file_name = "fasttext.bin"
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        if self.problem_type not in (BINARY, MULTICLASS):
```

Comment:
For now, can this be moved to _fit? We want to avoid the models crashing on init, as they aren't try/except handled at that point.